turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -103,18 +103,43 @@ impl ModuleResolveResultItem {
     }
 }
 
+#[derive(
+    Debug,
+    Default,
+    Clone,
+    Hash,
+    TraceRawVcs,
+    Serialize,
+    Deserialize,
+    Eq,
+    PartialEq,
+    ValueDebugFormat,
+    NonLocalValue,
+    TaskInput,
+)]
+pub enum Export {
```

Comment:
Should this be called `ExportUsage`?   since it is really about how a module is used in the context of an `importing` file...

I also wonder if, eventually perhaps, we could model `import * as ns from ...';....  ns.foo` as a named export usage of `foo` (and in particular not the other exports from that `ns`)

Reply:
I renamed it, and about the second comment, the analyzer already resolves `ns.foo` as a named export of `foo` so the `ExportUsage` would be `ExportUsage::Named("foo")`